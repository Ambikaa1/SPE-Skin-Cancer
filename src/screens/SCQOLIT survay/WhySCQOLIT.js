import React from "react";
import {View, Text, StyleSheet, TouchableOpacity, TextInput, ScrollView, Linking, Alert} from "react-native";


const WhySCQOLIT = ({navigation}) =>{
    return(
        <ScrollView style = {styles.container}>
            <View>
                <Text style = {styles.title}>
                    Why should I complete the SCQOLIT?
                </Text>
            </View>
            <View>
                <Text style ={styles.text}>
                    This confidential survey was developed to assess how skin cancer affects your quality of life.
                </Text>

                <Text style = {styles.text}>
                    It assesses how you are coping with your diagnosis.  You have the option of sharing it with your clinical team.  It can help them identify whether you would benefit from extra support.
                </Text>
            </View>
            <View style ={styles.questionView}>
                <Text style = {styles.question}>
                    Would you like to complete the SCQOLIT questions today?
                </Text>
            </View>
            <TouchableOpacity style = {styles.yesButton} onPress ={() => navigation.navigate("InfoSCQOLIT")}>
                <Text style = {styles.yesText}>
                    Yes
                </Text>
            </TouchableOpacity>
        </ScrollView>
    );
};

const styles = StyleSheet.create({
    container: {
        flex: 1,

    },
    title:{
        fontWeight: "bold",
        marginTop: 10,
        marginBottom: 60,
        fontSize: 30,
        textAlign:'center',
        },
    text:{
        marginBottom: 50,
        textAlign: 'center',
        fontSize:20,
        alignItems: 'center',
    },
    yesButton:{
        backgroundColor: "#24b324",
        alignItems: "center",
        borderRadius: 10,
        marginLeft:10,
        marginTop: 40,
        width: "95%",
    },
    yesText:{
        fontSize: 30,
        fontWeight: "bold",
        color: "white",
        marginVertical: 10,
    },
    questionView:{
        alignItems:'center',
        marginTop: 60,
        marginLeft:20,
        marginRight:20,
    },
    question:{
        fontWeight: "bold",
        fontSize: 25,
        textAlign: 'center',
    }


})
export default WhySCQOLIT;
