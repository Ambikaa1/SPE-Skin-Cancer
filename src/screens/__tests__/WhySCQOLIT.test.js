import React from 'react';
import renderer from 'react-test-renderer';

import WhySCQOLIT from "../WhySCQOLIT";

describe('<WhySCQOLIT />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<WhySCQOLIT />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
