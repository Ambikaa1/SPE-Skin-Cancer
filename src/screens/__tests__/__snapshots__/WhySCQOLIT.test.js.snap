// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WhySCQOLIT /> renders correctly 1`] = `
<RCTScrollView
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View>
    <View>
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "bold",
            "marginTop": 10,
            "paddingBottom": 20,
            "textAlign": "center",
          }
        }
      >
        Why should I complete the SCQOLIT?
      </Text>
    </View>
    <View>
      <Text
        style={
          Object {
            "alignItems": "center",
            "fontSize": 17,
            "marginBottom": 30,
            "textAlign": "center",
          }
        }
      >
        This confidential survey was developed to assess how skin cancer affects your quality of life.
      </Text>
      <Text
        style={
          Object {
            "alignItems": "center",
            "fontSize": 17,
            "marginBottom": 30,
            "textAlign": "center",
          }
        }
      >
        It assesses how you are coping with your diagnosis.  You have the option of sharing it with your clinical team.  It can help them identify whether you would benefit from extra support.
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#71A1D1",
          "borderRadius": 10,
          "marginHorizontal": 10,
          "marginTop": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 30,
            "fontWeight": "bold",
            "marginVertical": 10,
          }
        }
      >
        Next
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
